src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -8205,6 +8205,14 @@ private static Value expected_damage(
         1.0 - (Math.sqrt(Math.min(1000, KoLCharacter.getDamageAbsorption()) / 10.0) - 1.0) / 10.0;
     double elementAbsorb =
         1.0 - KoLCharacter.getElementalResistance(monster.getAttackElement()) / 100.0;
+
+    // https://kol.coldfront.net/thekolwiki/index.php/Ninja_snowman_assassin
+    if (monster.getName().equals("ninja snowman assassin")) {
+      baseValue = Math.max(0, attack - defenseStat) + 120;
```

Comment:
There is no point doing the initial calculation in 8201 if you are going to override it here. Declare the variables ahead of time (`int baseValue; double elementAbsorb`) then do `if...else`